argo_qa/solrconfig.xml

```diff
@@ -33,73 +27,150 @@
       <int name="qs">1</int>
       <int name="ps">2</int>
       <float name="tie">0.01</float>
-      <!-- this qf and pf are used by default, if not otherwise specified by
-           client. The default blacklight_config will use these for the
-           "keywords" search. See the author_qf/author_pf, title_qf, etc
-           below, which the default blacklight_config will specify for
-           those searches. You may also be interested in:
-           http://wiki.apache.org/solr/LocalParams
-      -->
-       <str name="qf">
-         id
-         all_text_timv
-         active_fedora_model_ssi
-         object_type_si
-       </str>
-       <str name="pf">
-         all_text_timv^10
-       </str>
```

Comment:
changed these to potentially existing fields;  note that argo currently passes in these params via blacklight_config_helper